lib/views/introduction/slides/fpas_server_select.dart

```diff
@@ -1,164 +1,32 @@
-      _serverURLTextfieldController.text = widget.selectedServerSettings!.url;
-    }
-  }
-
-  @override
-  void dispose() {
-    _serverURLTextfieldController.dispose();
-
-    super.dispose();
-  }
-
-  @override
-  Widget build(BuildContext context) {
-    void onServerOptionSelected(bool? option) {
-      isCustomServerSelected = option ?? false;
-
-      setState(() {});
-    }
-
-    Future<void> fetchServerSettings({String? newUrl}) async {
-      var url = newUrl ?? constants.defaultFPASServerUrl;
-      ServerSettings? serverSettings;
-      try {
-        serverSettings = await ref
-            .read(alertApiProvider)
-            .fetchServerSettings(overrideUrl: url);
-      } on UnreachableServerError {
-        serverSettings = null;
-      } on ArgumentError {
-        serverSettings = null;
-      }
-
-      serverSettingsConfirmed = newUrl != null ? false : serverSettings != null;
-      isServerURLValid = serverSettings != null;
-
-      if (serverSettings != null) {
-        widget.onServerSelected(serverSettings);
-      }
-
-      setState(() {});
-    }
-
-    Future<void> onSave() async {
-      if (!_formKey.currentState!.validate()) {
-        serverSettingsConfirmed = false;
-        isServerURLValid = false;
-
-        setState(() {});
-        return;
-      }
-
-      await fetchServerSettings(newUrl: _serverURLTextfieldController.text);
-
-      // unset focus to hide the keyboard again
-      FocusManager.instance.primaryFocus?.unfocus();
-    }
-
-    return _FPASServerSlideLayout(
-      isCustomServerSelected: isCustomServerSelected,
-      serverSettingsConfirmed: serverSettingsConfirmed,
-      isServerURLValid: isServerURLValid,
-      serverSettings: widget.selectedServerSettings,
-      formKey: _formKey,
-      serverURLTextfieldController: _serverURLTextfieldController,
-      onServerOptionSelected: onServerOptionSelected,
-      onDefaultServerConfirmPressed: fetchServerSettings,
-      onSave: onSave,
-    );
-  }
-}
-
-class _FPASServerSlideLayout extends StatelessWidget {
-  const _FPASServerSlideLayout({
-    required this.isCustomServerSelected,
-    required this.serverSettingsConfirmed,
-    required this.isServerURLValid,
-    required this.serverSettings,
-    required this.formKey,
-    required this.serverURLTextfieldController,
-    required this.onServerOptionSelected,
-    required this.onDefaultServerConfirmPressed,
-    required this.onSave,
-  });
+import 'package:foss_warn/services/url_launcher.dart';
 
-  final bool isCustomServerSelected;
-  final bool serverSettingsConfirmed;
-  final bool? isServerURLValid;
-  final ServerSettings? serverSettings;
-  final GlobalKey<FormState> formKey;
-  final TextEditingController serverURLTextfieldController;
-  final void Function(bool? option) onServerOptionSelected;
-  final VoidCallback onDefaultServerConfirmPressed;
-  final VoidCallback onSave;
+class IntroductionFPASServerSelectionSlide extends StatelessWidget {
```

Comment:
Since this page is not about selecting the server any more but rather informing the user about it, please rename the widget to indicate as such.